### DISCUSSION TITLE
Time Picker in android

### DISCUSSION THREAD

**Discussion by Lakshmanan4233** [appium/appium#18893] [Q&A] [answered]
How to select the time and hour in the UI? Is this possible 
I can get the elements for time stamp only . If the i click the clock circle area , then elements not visible . It is showing only class name. How to do it with  that?
![Snag_7d5f37](https://github.com/appium/appium/assets/96466534/c4ecaff5-51e8-4d92-bef5-6a6415ab2af0)

**Comment by jlipps** [comment, marked as answer]
It seems that this custom view does not support any accessibility labels on internal elements. You could try to `sendKeys` on the slider element itself, but other than that you're probably out of luck and you'd need to make a request to the developer of the view itself.

**Reply by Lakshmanan4233** [reply]
It worked using sendkeys . Thank you .
